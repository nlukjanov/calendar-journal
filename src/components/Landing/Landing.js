import React from 'react';

const Landing = () => {
  return <div data-testid='landing'>Hello</div>;
};

export default Landing;