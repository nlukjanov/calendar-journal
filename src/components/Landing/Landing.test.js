import React from 'react';
import { BrowserRouter } from 'react-router-dom';
// eslint-disable-next-line node/no-unpublished-import
import { render, waitFor } from '@testing-library/react';
import Landing from './Landing';

test('renders welcome on landing page', async () => {
  const { getByText } = render(
    <BrowserRouter>
      <Landing />
    </BrowserRouter>,
  );
  await waitFor(() =>
    expect(getByText(/Welcome to Calendar Journal/i)).toBeInTheDocument(),
  );
  expect(
    getByText('Take your notes and keep your memories'),
  ).toBeInTheDocument();
});
