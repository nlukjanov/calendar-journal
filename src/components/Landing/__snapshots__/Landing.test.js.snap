// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders welcome on landing page 1`] = `
<div>
  <div
    class="jumbotron"
  >
    <div
      class="text-center container"
    >
      <h1>
        Welcome to Calendar Journal
      </h1>
      <p>
        Take your notes and keep your memories
      </p>
      <div
        class="d-flex justify-content-center"
      >
        <a
          class="nav-link"
          href="/signin"
        >
          <button
            class="btn btn-primary"
            type="button"
          >
            Sign In
          </button>
        </a>
        <a
          class="nav-link"
          href="/signup"
        >
          <button
            class="btn btn-primary"
            type="button"
          >
            Sign Up
          </button>
        </a>
      </div>
    </div>
  </div>
</div>
`;
